Firmware/cmdqueue.cpp

```diff
@@ -91,14 +95,20 @@ bool cmdqueue_pop_front()
 
 void cmdqueue_reset()
 {
-    bufindr = 0;
-    bufindw = 0;
-    buflen = 0;
+	while (buflen)
+	{
+		uint8_t cmdqueue_current_type = *(cmdbuffer+bufindr);
```

Comment:
During compiling with PF-build.sh I get is that correct? That is a new warning.
```
/Prusa-Firmware/Prusa-Firmware-build/sketch/cmdqueue.cpp:100:11: warning: unused variable 'cmdqueue_current_type' [-Wunused-variable]
   uint8_t cmdqueue_current_type = *(cmdbuffer+bufindr);
           ^
```

Reply:
Ah, yes. Forgot about that one. It served a different purpose earlier, but then it was used only for the debugging line. Forgot to comment it out as well. Will probably insert it into the debug line.